jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/MetaInfConfiguration.java

```diff
@@ -767,6 +714,17 @@ protected List<Resource> findExtraClasspathDirs(WebAppContext context)
             .collect(Collectors.toList());
     }
 
+    private Resource ensureJarsAreOpened(WebAppContext context, Resource resource)
```

Comment:
Not sure about this method's name. At the very least, it should have a singular name as it only deals with one resource.